src/cordova/android/import_messages.action.mjs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from 'url';
+import {getRootDir} from '../../build/get_root_dir.mjs';
+import {readFile, readdir, writeFile, mkdir, access} from 'fs/promises';
+import {getNativeAndroidMessageDirectory} from './get_native_android_message_directory.mjs';
+import XML from 'xmlbuilder2';
+
+const ANDROID_XML_STRING_ID_PROPERTY = '@name';
+const ANDROID_XML_TEXT_CONTENT = '#';
+
+export async function main() {
```

Comment:
It's hard to tell what this program is doing. A comment and perhaps higher level functions would be helpful.